packages/edit-site/src/style.scss

```diff
@@ -25,6 +25,10 @@ body.toplevel_page_gutenberg-edit-site {
 	@include reset;
 }
 
+.components-modal__frame h1 {
```

Comment:
This is so the block styles don't leak into the saving modal (tried with TwentyTwenty that uses a distinct background-color).

Reply:
Added comment.